pages.it/linux/adduser.md

```diff
@@ -0,0 +1,27 @@
+# adduser
+
+> Servizio per aggiungere utenti.
+
+- Crea un nuovo utente con una home directory predefinita e richiede all'utente di impostare una password:
+
+`adduser {{nome_utente}}`
+
+- Crea un utente senza una home directory:
+
+`adduser --no-create-home {{nome_utente}}`
+
+- Crea un utente con una home directory nel percorso specificato:
+
+`adduser --home {{percorso/all/home}} {{nome_utente}}`
```

Comment:
I think this is a typo:
```suggestion
`adduser --home {{percorso/al/home}} {{nome_utente}}`
```

Reply:
The translated text is `percorso all'home` (I chose not to translate `home directory` as is quite specific)